GI05MOAS-GL537/GI05MOAS-GL537_D00003_ingest.csv

```diff
@@ -0,0 +1,5 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL537/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL537-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flord_m.flord_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL537/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL537-01-FLORDM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL537/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL537-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-GL537/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-GL537-04-CTDGVM000,telemetered,Available,
```

Comment:
looks good

Reply:
Looks good.  But there is not data in the Raw Data Repo.